operator/v1/types_ingress.go

```diff
@@ -125,6 +125,18 @@ type IngressControllerSpec struct {
 	//
 	// +optional
 	NodePlacement *NodePlacement `json:"nodePlacement,omitempty"`
+
+	// disableNamespaceOwnershipCheck disables the namespace ownership checks for
+	// a route host with different paths or for overlapping host names in the case
+	// of wildcard routes. Please be aware that if namespace ownership checks are
+	// disabled, routes in a different namespace can use this mechanism to 'steal'
+	// sub-paths for existing domains. This is only safe if route creation
+	// privileges are restricted, or if all the users can be trusted.
+	//
+	// If unset, the default is false
+	//
+	// +optional
+	DisableNamespaceOwnershipCheck *bool `json:"disableNamespaceOwnershipCheck,omitempty"`
```

Comment:
pointer based bool seems weird :P with nil being false..

Reply:
I [see this](https://github.com/openshift/api/pull/416#issuecomment-523658482) as part of a route admission policy struct with an enum for policy strings (or union types, which might be overkill)